### ISSUE TITLE
Add support for FTB modpacks

### ISSUE THREAD

**Issue by Zajozor** [sammko/picomc#3]
Currently you can successfully install modpacks from curseforge.
Some modpacks however are dropping support for curse and moving to the ftb launcher only instead.
(likely cause curse is not very nice to the devs)
Eg. see the note at the top of https://www.curseforge.com/minecraft/modpacks/ftb-interactions

Moved to FTB https://www.feed-the-beast.com/modpack/ftb_interactions
This site has also some nice stats https://ftb.neptunepowered.org/pack/ftb-interactions/

I did not really do the research whether some secret url can be mined out from the ftb site, which would have the same format as curse (or whether they require authentication).

If it is a pain to do, could you please leave at least some leads for me? I might consider opening a PR for it in the future..

**Comment by sammko** [comment]
The API used for FTB modpacks (api.modpacks.ch) is a beauty compared to curseforge. The functionality has been implemented in de32159.
```
Usage: picomc mod pack ftb install [OPTIONS] PACK_ID [VERSION]

  Install an FTB modpack.

  An instance is created with the correct version of forge selected and all
  the mods from the pack installed.

  PACK_ID can be the numeric id of the FTB modpack or the slug from the URL
  to its website.

  VERSION is the version name, for example 2.1.3, not its ID. If VERSION is
  not specified, the latest is automatically chosen. If --beta is used, the
  chosen version can be a beta version. Otherwise, only stable versions are
  considered.

Options:
  -n, --name TEXT  Name of the resulting instance
  -b, --beta       Consider beta modpack versions
  --help           Show this message and exit.
```

Please test it and close this issue if everything is in order. Version 0.4.1 has been released with this included.

**Comment by Zajozor** [comment]
Works, like a charm, thanks for the quick response!
